capstone/fitness/src/main/java/com/google/sps/servlets/CalendarEventsServlet.java

```diff
@@ -13,35 +13,28 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
+import com.google.api.client.util.DateTime;
 
-import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import java.io.FileNotFoundException;
 import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.security.GeneralSecurityException;
-import java.util.Collections;
-import java.util.List;
+
 import com.google.gson.Gson;
 import java.util.Date;
 
-/** Servlet that will deal with calendar. */
-@WebServlet("/calendar-servlet")
-public class CalendarServlet extends HttpServlet {
+// servlet that sends the date to calendar.html. 
```

Comment:
Please look through your files for correct comment formatting: capitaliziation, punctation, spacing.

Reply:
Okay.